Patch makes sure that the header files in included in installation
Author: Lara Peeters
diff -ruN pystring-1.1.4.orig/CMakeLists.txt pystring-1.1.4/CMakeLists.txt
--- pystring-1.1.4.orig/CMakeLists.txt	2022-09-27 02:28:46.000000000 +0200
+++ pystring-1.1.4/CMakeLists.txt	2025-08-06 17:37:10.073204631 +0200
@@ -17,6 +17,12 @@
 include(GNUInstallDirs)
 
 install(TARGETS pystring
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}
+)
+
+install(FILES pystring.h
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME}
+    COMPONENT developer
 )
 
